Makefile

```diff
@@ -34,7 +34,7 @@ generate: ## Generate protobuf stubs.
 
 .PHONY: build
 build: $(BUILD_DIR) ## Build go binary.
-	go build -ldflags "-w -X \"github.com/maticnetwork/polygon-cli/cmd/version.Version=dev ($(GIT_SHA))\"" -o $(BUILD_DIR)/$(BIN_NAME) main.go
+	go build -ldflags "-X \"github.com/maticnetwork/polygon-cli/cmd/version.Version=dev ($(GIT_SHA))\"" -o $(BUILD_DIR)/$(BIN_NAME) main.go
```

Comment:
`-w` is being dropped in this target? Any reason. I saw in your message that `-w` is in `LD_FLAGS` but that variable isn't used in this target.

Reply:
You are right. My bad.
Reverting this change.